packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -183,21 +197,35 @@ class MonacoUtils {
     log.debug2('monaco theme: ', MonacoTheme);
     monaco.editor.setTheme('dh-dark');
 
-    registerLanguages([DbLang, PyLang, GroovyLang, LogLang, ScalaLang]);
+    registerLanguages(([
+      DbLang,
+      PyLang,
+      GroovyLang,
+      LogLang,
+      ScalaLang,
+    ] as unknown[]) as Language[]);
```

Comment:
Instead of doing this - in the language files, declare the language like `const lang: Language = { id, conf, language }; export default lang`.

Will need to move the Language type to another file (probably just Language.ts)

Then you shouldn't need to cast here, it'll just be a `Language[]`.

Reply:
`const lang: Language = { id, conf, language };` 
this line causes lots of type errors. For example, the `Language` type require `blockComment` property to be of type `[string, string]`, but the language has type string[] for `blockComment`. Using as const also doesn't work since `Language` requires the array to be mutable. I think the easiest solution is to cast it

Reply:
What's the issue with `blockComment`? All of the `blockComment` we defined are of type `[string, string]`